### ISSUE TITLE
Reach out to localbitcoins experienced traders (kudos to @flix1)

### ISSUE THREAD

**Issue by ripcurlx** [bisq-network/growth#7]
## Learning Goal
Are we able to convince localbitcoins pro traders to also trade on Bisq?

## Hypothesis
By personally contacting localbitcoins pro traders, we are able to convince at least 5 of them to place offers and try Bisq out for a few weeks.

## Metric
As this is not easy to measure exactly, it will be more an monitoring of the platform volume and the direct communication with the pro traders on their experience.

## Experiment
Starting with BTCEUR, we will contact localbitcoins pro traders (https://localbitcoins.com/buy-bitcoins-online/eur/sepa-eu-bank-transfer/) personally (WhatsApp, Phone) to try out Bisq.

Points to discuss:

- _Fees & Deposit_: We need a good way to communicate our current fee and deposit structure and how it compares to LocalBitcoins fees (https://localbitcoins.com/fees, https://localbitcoins.com/blog/new-bitcoin-network-fee-model/). Regarding transaction/deposit fees we might be competitive, when we also take into account that the pro traders also use the LocalBitcoins advertising options a lot.
- _Spread_: The spread fluctuates a lot on Bisq at the moment based on the  number of trades and the market situation. 
- _Volume_: This is what we want the pro traders for
- _Ease of Trading_: We don’t need KYC and if you have made the account/wallet setup and clicked through all popups once, taking and creating offers is quite quick. 
- _Privacy_: It can’t get more private than with Bisq
- _Further Incentives_: As it is their business model to trade and generate profits, doing it on Bisq doesn’t cost them anything extra and they could potential earn BSQ tokens on top of their regular profits. (lots of discussion on this topic already happened on the #growthhacking channel on Slack)

## Result
> _- What happened?_
> _- What data did we collect?_
> _- Anything unexpected?_

## Next Step
> _- Pivot or persevere?_
> _- Another experiment for this goal?_
> _- Do we need to clean up?_

**Comment by flix1** [comment]
I'll have a first draft of a pitch to LBC traders in a few days, share it on slack and incorporate feedback for next week's call.

**Comment by cbeams** [comment]
@flix1 agreed on today's growth call (#16) to pick this issue up. By next week's call (Thu the 20th), the plan is to have a draft of a "pitch" document for LocalBitcoins traders. A "call sheet" of sorts that we can work from when talking to them, detailing:
 - the advantages of Bisq, especially from the perspective of an LBC trader.
 - a simple set of instructions how to get started providing liquidity
 - how to contact the team for support and feedback (Slack, forum, growth calls, etc)

Please add any other items you think should be here; whatever you think might help @flix1 put this together.

**Comment by flix1** [comment]
**Pitch 1 - Speed**
Bisq is the only Bitcoin exchange where you can be trading in 10 minutes. No account needed. No waiting for verification. No ID required.
Private. Safe. Decentralized.   https://bisq.network/

**Pitch 2 - Privacy**
Bisq is the only Bitcoin exchange where your privacy is respected. No account needed. No ID required. No KYC.
Private. Safe. Decentralized. https://bisq.network/

**Pitch 3 - Safety**
Bisq is the only Bitcoin exchange where you control your funds all the time. No withdrawal delays. No blocked accounts. No exit scams. No Mt.Gox
Private. Safe. Decentralized. https://bisq.network/ 

**Pitch 4 - Censorship resistance**
Bisq is the only Bitcoin exchange that is censorship-resistant. It is a P2P network. It's open source. It's global.
Private. Safe. Decentralized. https://bisq.network/

**Pitch 5 - Fraud prevention**
Bisq is the only P2P Bitcoin exchange that protects buyers and sellers through it's smart contract system of security deposits. 
Private. Safe. Decentralized. https://bisq.network/ 

**Pitch 6 - Business Opportunity for Localbitcoin traders.**
Become a market maker on Bisq! There is demand for traders and liquidity providers for many currencies. Be the first to trade your currency and profit! 
If you have traded Bitcoin on Localbitcoins before, you have everything you need to get started on Bisq. https://bisq.network/

**Comment by ripcurlx** [comment]
To have it in one place, I've copied over the Localbitcoins traders suggestions from @flix1 from the slack #growth channel:

## Twitter
- https://twitter.com/thebitcoinking, https://localbitcoins.com/accounts/profile/McVillan/
- https://twitter.com/Mqrius, https://localbitcoins.com/accounts/profile/Mqrius/
- https://twitter.com/vnistor
- https://twitter.com/kryptokitty
- https://twitter.com/affordableweb

## LinkedIn
- https://www.linkedin.com/in/davidoff-in-b14760a3/
- https://www.linkedin.com/in/matthew-saucier-702881140/
- https://www.linkedin.com/in/oscar-rono-3a760416/
- https://www.linkedin.com/in/shubham-nawariya-571103135/
- https://www.linkedin.com/in/luk-const-115927a4/
- https://www.linkedin.com/in/leonard-guerrero-1275699b/
- https://www.linkedin.com/in/ryan-white-293354151/
- https://www.linkedin.com/in/jose-marquez-60b9b368/
- https://www.linkedin.com/in/jeanman/
- https://www.linkedin.com/in/nico-van-meurs-90a9aa6/
- https://www.linkedin.com/in/alexbarbershop/

## Localbitcoins.com
- https://localbitcoins.com/buy-bitcoins-online/eur/sepa-eu-bank-transfer/
